subsys/nrf_security/src/drivers/Kconfig

```diff
@@ -36,6 +36,11 @@ config PSA_CRYPTO_DRIVER_CRACEN
 	help
 	  PSA crypto driver for the CRACEN HW peripheral.
 
+config PSA_CRYPTO_DRIVER_IRONSIDE
```

Comment:
Just a note that it makes sense to not have the `_SE` suffix here since it could act as a wrapper for different platforms with differente `IronSide` variants.